posthog/session_recordings/session_recording_api.py

```diff
@@ -250,9 +247,20 @@ def stream_from(url: str, headers: dict | None = None) -> Generator[requests.Res
         session.close()
 
 
+class SnapshotsBurstRateThrottle(PersonalApiKeyRateThrottle):
+    scope = "snapshots_burst"
+    rate = "120/minute"
+
+
+class SnapshotsSustainedRateThrottle(PersonalApiKeyRateThrottle):
+    scope = "snapshots_sustained"
+    rate = "600/hour"
```

Comment:
These were somewhat arbitrarily chosen. I halved our standard personal api key rate limiting. Have you any better alternatives @pauldambra? Recall you have to hit the `/snapshots` api twice to get a single recording (once to get the sources and then again to get the data)

Reply:
that works for me, we can always increase them but harder to reduce them.
other than DDOS risk of any endpoint here we just want it to be painful if you try to use this as a general export mechanism... since we want folk to tell us they want that, so we get signal to build it 🥼